.gitignore

```diff
@@ -82,6 +82,7 @@ docs/*/pdfreactor-fonts.html
 #################
 *.egg-info
 .installed.cfg
+static/
```

Comment:
I'm still not understanding, this would ignore `./static/` in the project root, right? Gulp will be dumping files in `./cfgov/v1/jinja2/v1`, right, not the root?